recipes/mapserver/meta.yaml

```diff
@@ -0,0 +1,77 @@
+# NB: Lowercase!
+{% set name = "mapserver" %}
+{% set version = "7.6.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://download.osgeo.org/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b46c884bc42bd49873806a05325872e4418fc34e97824d4e13d398e86ea474ac
+
+build:
+  number: 0
+  # Any volunteers? :)
+  skip: True  # [win or osx]
+  run_exports:
+    # Presently libmapserver.so.2 is linked to libmapserver.so.x.y.z
+    - {{ pin_subpackage(name, max_pin='x.x.x') }}
+
+requirements:
+  build:
+    - python
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+    - pkg-config  # [not win]
+  host:
+    - python
+    - pip
+    - freetype
+    - geos =3.10.1
```

Comment:
Is there a reason to not pin these to the same version in run?

Reply:
I thought I hit problems, but will try it shortly, thanks